zstackctl/zstackctl/ctl.py

```
@@ -8985,6 +9005,15 @@ def chown_to_zstack():
                 info('change permission to user zstack')
                 shell('chown -R zstack:zstack %s' % os.path.join(ctl.zstack_home, '../../'))
 
+            if not need_download and new_war.path:
+                ret = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)